bciers/apps/reporting/src/tests/components/dataGrid/model/operations/operationColumns.test.tsx

```diff
@@ -151,7 +151,7 @@ describe("operationColumns function", () => {
     await user.click(screen.getByText("Continue"));
 
     expect(useRouter().push).toHaveBeenCalledWith(
-      `reports/${row.report_version_id}/review-operator-data`,
+      `reports/${row.report_version_id}/review-operator-data?reports_title=${row.name}`,
```

Comment:
As much as possible Iike to hardcode the expected values in tests. This way we make the test slightly more sensitive to a change in string format, or catch side-effects transforming the initial value we pull data from.

```suggestion
      `reports/15/review-operator-data?reports_title=Operation with report`,
```